src/EnergyPlus/PollutionModule.hh

```diff
@@ -77,336 +140,160 @@ namespace PollutionModule {
+        PollFuel::Electricity
+    };
 
-        // Member Constructor
-        CoefficientProps(int const FuelFactorType,
-                         bool const FuelFactorUsed,
-                         Real64 const Source,
-                         Real64 const CO2,
-                         Real64 const CO,
-                         Real64 const CH4,
-                         Real64 const NOx,
-                         Real64 const N2O,
-                         Real64 const SO2,
-                         Real64 const PM,
-                         Real64 const PM10,
-                         Real64 const PM25,
-                         Real64 const NH3,
-                         Real64 const NMVOC,
-                         Real64 const Hg,
-                         Real64 const Pb,
-                         Real64 const Water,
-                         Real64 const NucHi,
-                         Real64 const NucLo,
-                         int const SourceSched,
-                         int const CO2Sched,
-                         int const COSched,
-                         int const CH4Sched,
-                         int const NOxSched,
-                         int const N2OSched,
-                         int const SO2Sched,
-                         int const PMSched,
-                         int const PM10Sched,
-                         int const PM25Sched,
-                         int const NH3Sched,
-                         int const NMVOCSched,
-                         int const HgSched,
-                         int const PbSched,
-                         int const WaterSched,
-                         int const NucHiSched,
-                         int const NucLoSched)
-            : FuelFactorType(FuelFactorType), FuelFactorUsed(FuelFactorUsed), Source(Source), CO2(CO2), CO(CO), CH4(CH4), NOx(NOx), N2O(N2O),
-              SO2(SO2), PM(PM), PM10(PM10), PM25(PM25), NH3(NH3), NMVOC(NMVOC), Hg(Hg), Pb(Pb), Water(Water), NucHi(NucHi), NucLo(NucLo),
-              SourceSched(SourceSched), CO2Sched(CO2Sched), COSched(COSched), CH4Sched(CH4Sched), NOxSched(NOxSched), N2OSched(N2OSched),
-              SO2Sched(SO2Sched), PMSched(PMSched), PM10Sched(PM10Sched), PM25Sched(PM25Sched), NH3Sched(NH3Sched), NMVOCSched(NMVOCSched),
-              HgSched(HgSched), PbSched(PbSched), WaterSched(WaterSched), NucHiSched(NucHiSched), NucLoSched(NucLoSched)
-        {
-        }
+    constexpr std::array<PollFuelComponent, (int)PollFuel::Num> pollFuel2pollFuelComponent = {
+        PollFuelComponent::Electricity,
+        PollFuelComponent::NaturalGas,
+        PollFuelComponent::FuelOil1,
+        PollFuelComponent::FuelOil2,
+        PollFuelComponent::Coal,
+        PollFuelComponent::Gasoline,
+        PollFuelComponent::Propane,
+        PollFuelComponent::Diesel,
+        PollFuelComponent::OtherFuel1,
+        PollFuelComponent::OtherFuel2,
+    };
+                
+    enum class PollFacilityMeter {
```

Comment:
More enumerations because each one is ever so slightly different. This one includes both Electricity components and District.